docs/build/nodes/collator/requirements.md

```diff
@@ -69,9 +69,9 @@ To run a collator, it is absolutely necessary to use a **CPU of minimum 4 Ghz pe
 | Component | Requirement |
 |---|---|
 | System | Ubuntu 20.04 |
```

Comment:
isn't better to change OS to ubuntu 22.04? @bLd75

Reply:
Right, updated